javascript-practice/src/js/controllers/userController.js

```diff
@@ -21,6 +21,9 @@ export default class UserController {
 
       if (userData) {
         this.view.redirectPage('dashboard.html');
+        localStorage.setItem('firstName', userData.firstName);
```

Comment:
Pls create constant for key of local storage. Ex: `FIRST_NAME: "firstName"`

Reply:
i done